apis/stargate/v1alpha1/stargate_types.go

```diff
@@ -188,6 +189,16 @@ type StargateSpec struct {
 	// +optional
 	// +kubebuilder:default=true
 	Auth *bool `json:"auth,omitempty"`
+
+	// Internode encryption stores which are used by Cassandra and Stargate.
+	// These should not be set explicitly by the user, they will be copied from the K8ssandraCluster object.
+	// +optional
+	ServerEncryptionStores *encryption.EncryptionStores `json:"server_encryption_stores,omitempty"`
```

Comment:
Similar to my previous comment, it would be nice to make these camel case to keep naming conventions more consistent.

Reply:
👍